src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -6,10 +6,11 @@
 public class CliSyntax {
 
     /* Prefix definitions */
-    public static final Prefix PREFIX_NAME = new Prefix("n/");
-    public static final Prefix PREFIX_PHONE = new Prefix("p/");
-    public static final Prefix PREFIX_EMAIL = new Prefix("e/");
-    public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
-    public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_NAME = new Prefix("-n ");
+    public static final Prefix PREFIX_PHONE = new Prefix("-p ");
+    public static final Prefix PREFIX_EMAIL = new Prefix("-e ");
+    public static final Prefix PREFIX_ADDRESS = new Prefix("-a ");
+    public static final Prefix PREFIX_RELATIONSHIP = new Prefix("-rs ");
+    public static final Prefix PREFIX_TAG = new Prefix("-t ");
```

Comment:
Perhaps we can consider removing `PREFIX_TAG` if it's no longer being used in any commands

Reply:
Yup, I agree. I can do this in a later PR.